doc/api/cluster.markdown

```diff
@@ -2,16 +2,45 @@
 
     Stability: 2 - Stable
 
-A single instance of Node.js runs in a single thread. To take advantage of
-multi-core systems the user will sometimes want to launch a cluster of Node.js
-processes to handle the load.
+All JavaScript running within a single instance of Node.js executes within the
```

Comment:
The first sentence is a bit long in the tooth and hard to understand. Maybe the same thing can be said with less.

"Node.js executes JavaScript code within the context of a single-threaded event loop."